frontend/App.js

```diff
@@ -1,19 +1,10 @@
-import React from 'react';
```

Comment:
Can't seem to get the viz to render on my Android or web, might need to look into that?

Reply:
It works on my iPhone (with the import fix mentioned below), but not on the web for me either. I'll look into it, but if we can't fix it by Saturday, we can always just demo it on an iPhone or an emulator.